### ISSUE TITLE
Bug: Overflow in Timer Input and Display Section on Smaller Devices

### ISSUE THREAD

**Issue by c-o-d-e-y-o-d-a** [CCExtractor/ultimate_alarm_clock#611]
### Description

When adding a timer and selecting the typing input method, an overflow occurs in the popup, causing layout issues and making the interface difficult to use. Additionally, after the timer is added, an overflow is also present in the section displaying the current timer. This issue is particularly noticeable on smaller devices, such as the Nexus 5. Addressing this will improve the usability and responsiveness of the timer functionality across all screen sizes.

### Screenshots

https://github.com/user-attachments/assets/22e8bed2-9639-4be2-bc21-8bba61d779c6

**Comment by khush1yaaar** [comment]
I am a proficient flutter developer, new to open source. please assign me this issue.

**Comment by AryanSarafDev** [comment]
@c-o-d-e-y-o-d-a  are you working on this?

**Comment by c-o-d-e-y-o-d-a** [comment]
> @c-o-d-e-y-o-d-a are you working on this?

yes, this and the other issue I raised, you can assign both to me.

**Comment by Vardhan-ysh** [comment]
Hi, can I be assigned to this issue?

**Comment by c-o-d-e-y-o-d-a** [comment]
> @c-o-d-e-y-o-d-a are you working on this?

yes
